src/main/java/com/minecolonies/entity/ai/minimal/EntityAICitizenAvoidEntity.java

```diff
@@ -34,8 +34,9 @@
      * @param farSpeed           how fast we should move when we are far away.
      * @param nearSpeed          how fast we should move when we are close.
      */
-    public EntityAICitizenAvoidEntity(EntityCitizen entity, Class<? extends Entity> targetEntityClass, float distanceFromEntity, double farSpeed, double nearSpeed)
+    public EntityAICitizenAvoidEntity(final EntityCitizen entity, final Class<? extends Entity> targetEntityClass, final float distanceFromEntity, final double farSpeed, final double nearSpeed)
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Split this 193 characters long line (which is greater than 180 authorized). [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AS00103)